<?xml version="1.0"?>
<launch>
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="1280" />
        <param name="image_height" value="720" />
        <param name="framerate" value="30" />
        <!-- <param name="camera_info_url" value="file://$(find docking)/config/cam_info.yaml" /> -->
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="usb_cam" />
    </node>
</launch>